contracts/reserve/GoodMarketMaker.sol

```diff
@@ -306,6 +306,22 @@ contract GoodMarketMaker is Initializable, DSMath, OwnableUpgradeable {
 		return gdReturn;
 	}
 
+	/**
+	* @dev Updates the bonding curve params by just increasing gdSupply but keeps reserveSupply same
+	* @param _gdAmount Amount of gd to add reserveParams
+	* @param _token The reserve token which is currently active
+	*/
+	function updateGdSupply(uint _gdAmount, ERC20 _token)
```

Comment:
this is not correct
either both  reservettoken + g$ token balances change
or the reserve ratio + one toke balance change